assets/training/finetune_acft_hf_nlp/environments/acpt/context/requirements.txt

```diff
@@ -14,3 +14,9 @@ rouge-score==0.1.2
 sacrebleu==2.2.1
 bitsandbytes==0.38.1
 einops==0.6.1
+cryptography==41.0.2
+scipy==1.10.0
+aiohttp===3.8.5
+certifi===2023.07.22
+torch==1.13.1
```

Comment:
Do we need these? Torch is already baked as part of acpt base image.

And we have following three afected packages: cryptography, scipy, numpy

Reply:
Shared on Teams. All the above pinned packages are affected.